src/components/AnimatedMapboxLayer.vue

```diff
@@ -181,6 +181,16 @@ export default class AnimatedMapboxLayer extends Vue {
     }
   }
 
+  setDoubleClickLayer() {
+    if (this.mapObject && this.mapObject.doubleClickZoom.isEnabled()) {
```

Comment:
Why is this a condition for enabling listening for double clicks?

Reply:
initially because of disabling the double click I guess. This is moved to another check inside this if

Reply:
The event used to be called multiple times when switching to a different data layer. This check makes it only be added the first time.